ecs-init/config/common.go

```diff
@@ -117,11 +122,14 @@ func DockerUnixSocket() (string, bool) {
 
 // getBaseLocationForRegion fetches the bucket URI from list of S3 Buckets by region name or default if key is not found
 func getBaseLocationForRegion(regionName string) string {
+	if regionName == RegionNameNotFound {
```

Comment:
I would expect that to be `config.DefaultRegionName` based on the version of the code that didn't have this superfluous `RegionNameNotFound`.  But yes, that looks like it models the failure to resolve a region from the metadata service.